mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactTextView.test.js

```diff
@@ -55,11 +55,55 @@ describe('ShowArtifactTextView', () => {
     const props = { ...minimalProps, getArtifact: getArtifact };
     wrapper = mount(<ShowArtifactTextView {...props} />);
     setImmediate(() => {
-      instance = wrapper.instance();
       wrapper.update();
       expect(wrapper.find('.ShowArtifactPage').length).toBe(1);
-      expect(wrapper.find('.text-area').length).toBe(1);
-      expect(wrapper.find('.text-area').prop('value')).toEqual('my text');
+      expect(wrapper.find('code').length).toBe(1);
+      expect(wrapper.find('code').text()).toBe('my text');
+      done();
+    });
+  });
+
+  test('should render a python script without crashing', (done) => {
+    const getArtifact = jest.fn((artifactLocation) => {
+      return Promise.resolve('print("foo")');
+    });
+    const props = { path: 'fake.py', runUuid: 'fakeUuid', getArtifact: getArtifact };
+    wrapper = mount(<ShowArtifactTextView {...props} />);
```

Comment:
I think if we use `shallow` to render the `ShowArtifactTextView`, we should actually be able to find a `SyntaxHighlighter` instance and assert on its props, e.g:

```
wrapper = shallow(<ShowArtifactTextView {...props} />);
setImmediate(() => {
  wrapper.update()
  expect(wrapper.find(SyntaxHighlighter).first().props().language).to.equal('py');
  done();
});
```

and same for the other tests (e.g. we can verify that MLproject and MLmodel files have yaml highlighting applied)

Reply:
Thanks! Fixed the tests.